File: API/BCMParkingAPI/BCMParkingAPI/Models/Member.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace BCMParkingAPI.Models
{
    public class Member
    {
        public Guid MemberId { get; set; }
        public Audit Audit { get; set; }
        public ContactInformation ContactInformation { get; set; }
    }
}

File: Android/BcmParking/app/src/main/java/com/ksubcm/parking/bcmparking/models/data/Member.java
package com.ksubcm.parking.bcmparking.models.data;

import io.realm.RealmObject;
import io.realm.annotations.PrimaryKey;

/**
 * Created by Stephen on 8/24/16.
 */

public class Member extends RealmObject {
    @PrimaryKey
    private String MemberId;
    public Audit Audit;
    public ContactInformation ContactInformation;
}
